linera-storage-service/src/client.rs

```diff
@@ -240,8 +240,20 @@ impl WritableKeyValueStore for ServiceStoreClientInternal {
             return Ok(());
         }
         let mut statements = Vec::new();
+        let mut full_key = self.start_key.clone();
+        full_key[0] = KeyTag::RootKey as u8;
+        let operation = Operation::Put(KeyValue {
+            key: full_key,
+            value: vec![],
+        });
```

Comment:
It may not be ideal to overwrite this key over and over. (E.g. it might evict other keys faster from any cache)
Would it be possible to only schedule one such write per connect?

Reply:
Yes, we can do that.
Only write at the first, this requires a `Arc<Mutex<bool>>` which I did for DynamoDb, but not for others as I thought it was needed for DynamoDb.